src/app/helpers/app-initializer/app-initializer.ts

```diff
@@ -80,7 +80,16 @@ export interface Keys {
  */
 export interface Configuration {
   kind: "Configuration";
+  /** Is the current environment running in production mode */
   production: boolean;
+  /** Timeout for web requests in server side renderer */
+  ssrTimeout: number;
+  /** Timeout for web requests in browser */
+  browserTimeout: number;
```

Comment:
nit: specifically _API_ requests only

Reply:
This affects all requests which go through the angular HttpClient library, does not specifically discriminate against API requests